### PR TITLE
mu-editor: Add version 1.0.2

### PR DESCRIPTION
**Pull request by niheaven**

### CODE REVIEW

**Comment by Ash258** [comment]
Does this work in all powershell versions? I think ps6 has AsByteStream instead od byte encoding

**Comment by niheaven** [comment]
For PS 6 & 7, it should use `-AsByteStream` here.

**Comment by Ash258** [comment]
Would not be more suitable something like this:

```powershell
if ($PSVersionTable.PSVersion.Major -gt 5) {
	$param = @{
		'AsByteStream' = $true
	}
} else {
	$param = @{
		'Encoding' = 'Byte'
	}
}

get-content $file @param
Set-content $file @param -nonewline
```

File: bucket/mu-editor.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        },
+        "32bit": {
+            "url": "https://github.com/mu-editor/mu/releases/download/1.0.2/mu-editor_1.0.2_win32.exe#/dl.7z",
+            "hash": "576baea0bd64649583a61bf6ce2836a91b02dce450c4b886d6adc7c35460ae8c"
+        }
+    },
+    "pre_install": [
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst*\" -Recurse -Force",
+        "if ((Get-Command Get-Content).Parameters.ContainsKey('AsByteStream')) {",
+        "   $enc = @{",
```
Comment (Ash258) [inline bucket/mu-editor.json]: You can keep initialization as one line.

```suggestion
        "   $enc = @{ 'AsByteStream' = $true }",
```

**Pull request merged**